nodes/Const.py

```diff
@@ -0,0 +1 @@
+x = 1
```

Comment:
Here I'd like to see just the expression `1` rather than an assignment statement.